### DISCUSSION TITLE
Kavita Update SCRIPT fails - release link? Not Found thank you

### DISCUSSION THREAD

**Discussion by cpcotton** [tteck/Proxmox#2953] [Help Needed] [answered]
/ //_/___ __   __(_) /_____ _
  / ,< / __ `/ | / / / __/ __ `/
 / /| / /_/ /| |/ / / /_/ /_/ / 
/_/ |_\__,_/ |___/_/\__/\__,_/  
                                
 |curl: (56) Recv failure: Connection reset by peer

[ERROR] in line 61: exit code 0: while executing command tar -xvzf <(curl -fsSL https://github.com/Kareadita/Kavita/releases/download/$RELEASE/kavita-linux-x64.tar.gz) --no-same-owner &> /dev/null

root@kavitaBooks17:~#

**Comment by tteck** [comment, marked as answer]
🏁 The script runs without any problems from this end.

```
    __ __            _ __       
   / //_/___ __   __(_) /_____ _
  / ,< / __ `/ | / / / __/ __ `/
 / /| / /_/ /| |/ / / /_/ /_/ / 
/_/ |_\__,_/ |___/_/\__/\__,_/  
                                
 ✓ Updated Kavita LXC
root@kavita:~# 
```

Sounds like network issues. 🤷🏻‍♂️
